core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -356,6 +357,8 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
   val serverChannel = openServerSocket(endPoint.host, endPoint.port)
   private val processors = new ArrayBuffer[Processor]()
   private val processorsStarted = new AtomicBoolean
+  private val idlePercentMeter = newMeter(s"Acceptor$IdlePercentMetricName",
```

Comment:
Do we really want a name with a `$`?

Reply:
@ijuma Thanks for the review. It is using string interpolation, metric name is `AcceptorIdlePercent`. Hope I haven't missed something.

Reply:
My bad, it just looked a bit odd in the diff.